Header header

float64[] timestamps
geometry_msgs/Vector3[] positions
geometry_msgs/Quaternion[] orientations
